docker-compose-prod.yml

```diff
@@ -10,12 +10,16 @@ services:
     container_name: "elasticsearch"
     environment:
       - discovery.type=single-node
+      - ELASTIC_PASSWORD=$ELASTIC_PASSWORD
+      - xpack.security.enabled=$ELASTIC_SECURITY
```

Comment:
The documentation says that this should be a boolean?

Reply:
The env variable is set on the server and is indeed a boolean